### PR TITLE
Improve wording of paper

### PR DESCRIPTION
**Pull request by DRMacIver**
Some minor wording tweaks to the JOSS paper.

### CODE REVIEW

**Comment by DRMacIver** [comment]
@danielskatz how does this sound as a fix in response to https://github.com/openjournals/joss-reviews/issues/1891#issuecomment-555499327? 

(It's a bit more intrusive than what you suggested because I realised the lead in there was also weirdly dangling)

**Comment by danielskatz** [comment]
lgtm

File: paper.md [hunk 1]
```diff
@@ -59,8 +59,9 @@ and has the potential to significantly improve the quality of the associated sci
 
 # Hypothesis for Software Testing Research
 
-This is true both because of the much wider array of software that can now easily be tested with property-based testing,
-and because it has a novel implementation that lifts a major difficulty that prior research on software testing faces.
+Hypothesis is a powerful platform for software testing research,
+both because of the wide array of software that can be easily tested with it,
+and because it has a novel implementation that solves a major difficulty that prior work in software testing research faced.
```
Comment (Zac-HD) [inline paper.md]: "solves a major difficulty faced by prior work..."?

Comment (danielskatz) [reply paper.md]: that's fine with me

**Pull request merged**